data/json/items/melee/bludgeons.json

```diff
@@ -483,7 +483,7 @@
     "volume": "1250 ml",
     "longest_side": "80 cm",
     "bashing": 50,
-    "to_hit": { "grip": "weapon", "length": "long", "surface": "any", "balance": "uneven" },
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "any", "balance": "uneven" },
```

Comment:
I may not have understood that category then, I assumed that since the grip was good and with the way it's hard to lose grip when using it properly and that they typically have a similar grip to axes they would be weapon grip. The axe is specifically called out as having a weapon grade grip.

Reply:
I'm going to ask actually, do you think axes should be moved to a solid grip? I personally think an axe grip is better than a sword or police baton. I could make all the regular tool handled 'weapons' as solid grip.

Reply:
`"grip": "weapon",      // one of "bad", "none", "solid", or "weapon"` Bad is chronically underused and weapon is heavily overused, imo.

Reply:
Sorry this took so long Maleclypse but I've been trying to formulate my thoughts in a coherent way. I reread through the documentation on grip and talked with several people on discord about this and I've definitely changed my mind. The "solid" grip group specifically calls out pipes and pool cues while the "none" group talks about 2x4s, TVs, and clothing. I definitely think that quartstaffs, axes, and spears would fit the former category better than the latter. When I talked to the others in discord I found out that many historical polearms are either oblong or octagonal shaped for better grip, and looking online I also saw that some are wrapped in string or the like for better grip as well. I think we either need to rewrite the standards for grip altogether or shift most basic polearms to be solid and then have ones that specifically mention the shape or the recipes implies wrapping be made "weapon" grip. One thing to keep in mind is that this file dedicated to mostly real weapons; length and striking surface change the way the weapon is used but having a dedicated weapon with a bad grip just doesn't make much sense.